src/storage/copier/table_copier.cpp

```diff
@@ -310,5 +311,14 @@ void TableCopier::throwCopyExceptionIfNotOK(const arrow::Status& status) {
     }
 }
 
+void TableCopier::initializePropertyIdxToBatchColumnIdxMap() {
+    auto propertyIdxes = getAllPropertyIdxes();
+    std::sort(propertyIdxes.begin(), propertyIdxes.end());
+    col_idx_t batchColumnIdx = getPropertyBatchColumnStartIdx();
```

Comment:
Is there a difference between `Column` and `BatchColumn`? If not I would just use Column for simplicity ( or FileColumn if you wanna differentiate with the column used in storage)